lib/libzutil/zutil_import.c

```diff
@@ -1128,7 +1126,7 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 	if (num_labels != NULL)
 		*num_labels = count;
 
-	free(labels);
+	umem_free_aligned(labels, PAGESIZE);
```

Comment:
Same here regarding the second parameter. It should be VDEV_LABELS * sizeof (*labels).